database/seeds.sql

```diff
@@ -0,0 +1,70 @@
+insert into dbo.dm_addresses (ref_addresses_people_id, ref_address_id, person_id, start_date, end_date, address, post_code, district, paf_authority, ward, is_in_la_area, address_type, is_contact_address, is_display_address, housing_tenure, is_hmo, oz_lga_name, oz_lga_number, oz_sla_id, oz_lhd_id, oz_hacc_id, oz_phams_id, access_notes, easting, northing, unique_id, from_dm_person) values (85, 15, 35, '1966-04-30T23:34:10Z', '2064-10-26T17:39:53Z', '00 Oneill Way', 'E8 2LJ', 'Lower Clapton', null, 'Dalston', 'Y', null, 'N', 'N', null, 'Y', null, 98, 48, 29, 71, 9, 'morbi quis tortor id nulla ultrices aliquet maecenas leo odio condimentum id luctus nec', 533950, 185102, 75, 'N');
+
+-- add test telephone numbers
+insert into dbo.dm_telephone_numbers (telephone_number_id, person_id, telephone_number, telephone_number_type) values (7, 78, '6903446318', 'Mobile');
+insert into dbo.dm_telephone_numbers (telephone_number_id, person_id, telephone_number, telephone_number_type) values (100, 21, '5087280941', 'Work');
+insert into dbo.dm_telephone_numbers (telephone_number_id, person_id, telephone_number, telephone_number_type) values (9, 21, '6344116355', 'Mobile');
+insert into dbo.dm_telephone_numbers (telephone_number_id, person_id, telephone_number, telephone_number_type) values (7, 78, '4489769930', 'Work');
+insert into dbo.dm_telephone_numbers (telephone_number_id, person_id, telephone_number, telephone_number_type) values (81, 70, '7712705477', 'House');
+insert into dbo.dm_telephone_numbers (telephone_number_id, person_id, telephone_number, telephone_number_type) values (18, 19, '4457745916', 'House');
+insert into dbo.dm_telephone_numbers (telephone_number_id, person_id, telephone_number, telephone_number_type) values (89, 88, '7326547202', 'Work');
+insert into dbo.dm_telephone_numbers (telephone_number_id, person_id, telephone_number, telephone_number_type) values (9, 71, '4691853823', 'Mobile');
+insert into dbo.dm_telephone_numbers (telephone_number_id, person_id, telephone_number, telephone_number_type) values (54, 19, '1717514648', 'Work');
+insert into dbo.dm_telephone_numbers (telephone_number_id, person_id, telephone_number, telephone_number_type) values (39, 97, '6466001065', 'House');
+
+-- add test teams
+insert into dbo.sccv_team (id, name, context) values (32, 'Rutrum', 'C');
+insert into dbo.sccv_team (id, name, context) values (15, 'Sodales', 'A');
+insert into dbo.sccv_team (id, name, context) values (74, 'Primis', 'C');
+insert into dbo.sccv_team (id, name, context) values (35, 'Aenean', 'C');
+insert into dbo.sccv_team (id, name, context) values (20, 'Tristique', 'C');
+insert into dbo.sccv_team (id, name, context) values (24, 'Lectus', 'A');
+
+-- add test workers
+insert into dbo.sccv_worker (id, email, first_name, last_name, role) values (21, 'zmctague0@eventbrite.com', 'Zebedee', 'McTague', 'est');
```

Comment:
Should we update the emails to use some fake domains, so it doesn't look like we are promoting something? These will also be visible in the app so might raise some questions there.

Reply:
Ideally we should use dummy domains. If we update this source and then update the existing records in the db, I think that should work. Then it would be fine moving forward.

Reply:
[http://example.com/](http://example.com/) is a good candidate?

Reply:
Yup, definitely! We'll change to all to `@example.com`.

Reply:
Updated in https://github.com/LBHackney-IT/social-care-case-viewer-api/pull/161/commits/cd4e39cfe7a04f930c5a3b7d7764ffb5af236990. We'll update the staging DB afterwards.